tests/storage-spec.js

```diff
@@ -155,7 +163,7 @@ describe('languageStorage factory', function () {
                     label: 'Polski',
                 },
                 {
-                    id: 'en_us',
+                    id: 'en',
```

Comment:
Btw I didn't mean that code literally; the `languageIdMatch` would be checking if the navigator language matches the choices before the code decides to check against the culture-stripped one.